File: 2.0/java/me/sendVerificationCodeToCallerId.java
import com.callfire.api.client.CallfireClient;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient callfireClient = new CallfireClient("api_login", "api_password");
        String phoneNumber = "12132224444";
        callfireClient.meApi().sendVerificationCode(phoneNumber);
    }
}

File: 2.0/csharp/me/sendVerificationCodeToCallerId.cs
using CallfireApiClient;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        const string phoneNumber = "12132000384";
        client.MeApi.SendVerificationCode(phoneNumber);
    }
}
